Gruntfile.cjs

```diff
@@ -25,6 +25,7 @@ module.exports = function(grunt) {
             cwd: './',
             src: [
               '*.html',
+              'background.js',
```

Comment:
part of the v3 manifest changes requires that `background.js` is in the project's root.